packages/daemon/test/test-endo.js

```diff
@@ -29,8 +30,8 @@ const locator = {
   cachePath: path.join(dirname, 'tmp/cache'),
   sockPath:
     process.platform === 'win32'
-      ? raw`\\?\pipe\endo-test.sock`
-      : path.join(dirname, 'tmp/endo.sock'),
+      ? raw`\\?\pipe\endo-test-${process.pid}.sock`
```

Comment:
I’m curious whether the problem can be better addressed by ensuring that Endo stomps over the existing PIDfile, since restarting without a graceful shutdown will affect end users as well.

Reply:
Or, if handling the named pipe gracefully on Windows is not in the cards, we may need to adjust some other behavior around `@endo/where` for discovering the rendezvous location of the Endo named pipe by looking first at the PID file to infer the PID in the named pipe.